### ISSUE TITLE
Client Allocation Request for: WillTechMusing

### ISSUE THREAD

**Issue by William8Work** [filecoin-project/filecoin-plus-client-onboarding#368]
---

# Client Allocation Questions 

## Core Information
- Name: WillTechMusing
- Website / Social Media: https://ctzero.com/
- Region: North America
- DataCap Requested: 5TiB
- Addresses to be Notarized: f17du2ulfhtpwgs4rwd4ngnrblueatjv4c5n57umy
- Notary Requested: andrewxhill

**Comment by William8Work** [comment]
Hi,

I saw you asked similar questions for other request so I am going ahead to provide you additional background:

1. yourself/ your organization (with links to official website/social media/news mentions etc)

This is a new startup web site to capture Innovation training videos and materials to help professional and corporate executives learn about ways and approach to make their organization more innovative. https://ctzero.com/

2. Your history working on IPFS/Filecoin (notable contributions to the community, type of engagement shown over time etc)

I am currently a miner but came from Technology Management and Consulting background. Would like to take the opportunity to introduce IPFS/Filecoin into the education/training industry.

3. Use case: type/source/quantity/diversity/scarcity of the data you plan to store on Filecoin

The training materials will come from partners (professors, successful corporate executives, advisors, etc) in the Innovation space. These materials will be developed and authorized by the partners. This data will be granted as public materials and make free to everyone on the topic of Innovation.

The quantity of the data are documents, sound tracks, videos from these authors about Innovation for education. Each author might provide GBs of data, so in 6-12 months would be 5TB+. 

4. Miners you intend to allocate DataCap with allocation proportion(with redundancy/location/specific requirement of storage miners)

Initially for North America market and Asia market so prefer miners in these geo locations.

5. What's your next plan for the data you plan to store in Filecoin Network? For-profit or not?

The data part are available for public and free. 

6. Would you be OK in posting deal transactions here?

Yes its OK.

**Comment by andrewxhill** [comment]
Hey, thanks for that, the questions I ask are a bit different but have some overlap

1. Can you introduce yourself? Where can I find you online? Examples include Github, Twitter, LinkedIn, WeChat, etc., and more than one is encouraged.
2. If you represent an organization, what is the name, nature, registration country, time since inception, and named officers of your organization?
3. In which region(s) do you intend to use the DataCap?
4. What is your use case for the DataCap being requested?
5. How many miners do you plan on allocating the DataCap? Please specify if you have a specific miner(s) you would like to allocate the DataCap.
6. Do you agree on providing transparency of DataCap allocation decisions & execution details in public to prevent excessive/unfair self-dealing? If so, how will you plan to report it?
7. Will your organization be responsible for allocating the full DataCap? Examples where this might not be the case, include app builders where the application end-users may have access to storage backed by the DataCap. Please read our full proposal for details on this approach. Unfortunately, our proposed multisig enabled DataCap provider is not ready today, but when it is, we will provider further documentation.

Feel free to shuffle around your original response and extend for these questions. I'd like to capture your responses in context since they may be used to evaluate future applications. 

Out of curiosity, the data that you are proposing to upload here, is that data you already have access to or is it something that you are planning to gather after having the datacap? It also sounds like it might have restrictive licensing?

**Comment by William8Work** [comment]
thank you @andrewxhill. 

1. Can you introduce yourself? Where can I find you online? Examples include Github, Twitter, LinkedIn, WeChat, etc., and more than one is encouraged.

my slack alias @WillTechMusing, Github account: https://github.com/William8Work, 

2. If you represent an organization, what is the name, nature, registration country, time since inception, and named officers of your organization?

This is a new startup web site to share Innovation training videos and materials to help professional and corporate executives learn about ways and approach to make their organization more innovative. https://ctzero.com/.

This is intended for non-profit and has not yet registered as formal corporation. 

3. In which region(s) do you intend to use the DataCap?

Mostly North America, and maybe some in Asia or Europe.

4. What is your use case for the DataCap being requested?

The training materials will come from partners (professors, successful corporate executives, advisors, etc) in the Innovation space. These materials will be developed and authorized by these partners. This data will be granted permission as public materials and make free to everyone on the topic of Innovation.

The quantity of the data are documents, sound tracks, videos from these authors about Innovation for education. Each author might provide GBs of data, so in 6-12 months would be 5TB+.


5. How many miners do you plan on allocating the DataCap? Please specify if you have a specific miner(s) you would like to allocate the DataCap.

I know a list of miners from MinerX program so they could be a good group to start with.


6. Do you agree on providing transparency of DataCap allocation decisions & execution details in public to prevent excessive/unfair self-dealing? If so, how will you plan to report it?

yes I agree. One way is to provide a monthly report on the list of miners with the deals and sizes as auditing. This can be shared with whoever might be interested in it.


7. Will your organization be responsible for allocating the full DataCap? Examples where this might not be the case, include app builders where the application end-users may have access to storage backed by the DataCap. Please read our full proposal for details on this approach. Unfortunately, our proposed multisig enabled DataCap provider is not ready today, but when it is, we will provider further documentation.

yes the datacap will be used to stored the materials (documents, audio and videos). And the decision will be made by this organization. 


8. Out of curiosity, the data that you are proposing to upload here, is that data you already have access to or is it something that you are planning to gather after having the datacap? It also sounds like it might have restrictive licensing?

I have access to some data already (less than 1TiB). Additional data will be gathered through showing cases the capabilities of the web site and need some marketing and partnerships. Because these data are authentic from the authors, there will be agreement from them to authorize the website to share these materials to the public.

**Comment by andrewxhill** [comment]
Based on those parameters, I can provide a maximum of 100GiB of datacap this time around. What I recommend if you need the full 5TiB is to use the 100Gib and generate an open record of use that will benefit a follow-up application. Let me know if the 100GiB will be useful to you, and we can finalize it.

**Comment by William8Work** [comment]
This is great. Yes I can definitely start with 100GiB and will generate an
open record of use per your suggestion.

Thank you Andrew and this is exciting!

**Comment by andrewxhill** [comment]
hey @William8Work 

we made an annoying mistake. in the step to verify that your address was able to accept the DataCap we used https://verify.glif.io. however, in submitting the check we submitted your address in the wrong field (the one that grants automated datacap)! now you have 32GiB Datacap at `f17du2ulfhtpwgs4rwd4ngnrblueatjv4c5n57umy ` instead. 

so, two options. go ahead and use the 32GiB with that address. or, submit a different address now and we can give you the proper 100GiB limit. 

let me know.
